src/myrtlespeech/model/rnn.py

```diff
@@ -146,59 +144,58 @@ def forward(self, x: Tuple[RNNData, Lengths]) -> Tuple[RNNData, Lengths]:
                 sequence input and the second represents the length of these
                 *input* sequences.
 
+            hx: The Optional hidden RNNState.
+
         Returns:
-            A Tuple[RNNData, Lengths] where the first element is the rnn
-                sequence output and the second represents the length of these
-                *output* sequences. These lengths will be unchanged from the
-                input lengths.
-
-                The sequence output of :py:obj:`RNNData` will have the same
-                subtype as ``x[0]`` so, if the user would like the hidden state
-                returned at the start-of-sequence, they should pass a hidden
-                state of :py:data:`None` and PyTorch will initialise the
-                hidden state(s) to zero.
+            A Tuple[Tuple[RNNData, Lengths], RNNState] where the first element
+                is the rnn sequence output and the second represents the
+                length of these *output* sequences. These lengths will be
+                unchanged from the input lengths.
```

Comment:
The `rnn sequence output` does imply hidden state but it may be a good idea to mention it more explicitly?

Reply:
Godd point- done!